server.js

```diff
@@ -6,6 +7,7 @@ const app = express();
 const PORT = 3000;
 
 app.use(express.static(path.resolve('dist')));
+app.use(fallback('index.html', { root: 'dist' }))
```

Comment:
Если было решено использовать в проекте `;`, то следует использовать везде
```suggestion
app.use(fallback('index.html', { root: 'dist' }));
```